mkosi/config.py

```diff
@@ -56,15 +56,17 @@ class Verb(StrEnum):
     help          = enum.auto()
     genkey        = enum.auto()
     documentation = enum.auto()
+    journalctl    = enum.auto()
+    coredumpctl   = enum.auto()
 
     def supports_cmdline(self) -> bool:
-        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.ssh)
+        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.ssh, Verb.journalctl, Verb.coredumpctl)
 
     def needs_build(self) -> bool:
-        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.serve)
+        return self in (Verb.build, Verb.shell, Verb.boot, Verb.qemu, Verb.serve, Verb.journalctl, Verb.coredumpctl)
 
     def needs_root(self) -> bool:
-        return self in (Verb.shell, Verb.boot)
+        return self in (Verb.shell, Verb.boot, Verb.journalctl, Verb.coredumpctl)
```

Comment:
Do we need root if we're invoking this on a directory image?

Reply:
I'm not sure, I'm just going for lunch and will test in an hour or so.

Reply:
Nope it doesn't, I have added a check as early as possible for it here: https://github.com/CodethinkLabs/mkosi/blob/bb63b8d74e4379f526fe0093de9370ffed270dc1/mkosi/__init__.py#L2591